package.json

```
@@ -30,7 +30,7 @@
     "@dashevo/dapi-client": "~0.9.0-dev.6",
```

0.9.0-dev was released as  0.11.0 (to align platform component versions). Do we want to move to the released version or are there conflicts?
```suggestion
    "@dashevo/dapi-client": "~0.11.0",
```